### PR TITLE
Task-44481: Long names of spaces are incorrectly displayed

### PR DESCRIPTION
**Pull request by wassimmhamdi**
Problem: The name of the space is displayed throughout the screen
Fix: Added ellipses to the space name when it is too long

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue [hunk 1]
```diff
@@ -20,7 +20,9 @@
         <v-list-item-title class="text-wrap">
           <template v-if="enabledDigestLabel">
             <v-chip class="ma-2" color="primary">
-              {{ enabledDigestLabel }}
+              <span :class="{ 'mobile-chip-ellipsis' : isMobile }">
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue]: Can you explain this modification ? Why we need to add the isMobile ?

**Review by rdenarie — CHANGES_REQUESTED** [review]
Please rebase the PR and resolve the conflict

File: webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue (hunk 1 above)
Comment (wassimmhamdi) [reply webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue]: Problem fixed

Comment (wassimmhamdi) [reply webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue]: it was a problem of updating in the branch

**Review by rdenarie — APPROVED** [review]

**Pull request merged**